.github/workflows/CI.yml

```diff
@@ -143,6 +144,15 @@ jobs:
           name: ${{ steps.ray_julia.outputs.tarball_filename }}
           path: ${{ steps.ray_julia.outputs.tarball_path }}
 
+      - name: Publish Pre-Release
+        uses: softprops/action-gh-release@v1
```

Comment:
Looks like you need to add additional permissions for our job: https://github.com/softprops/action-gh-release#permissions

Reply:
really? how come? it seems to have worked https://github.com/beacon-biosignals/Ray.jl/releases/tag/v0.0.2-dev

Reply:
That's surprising